examples/awssns-quickstart/src/main/resources/META-INF/microprofile-config.properties

```diff
@@ -0,0 +1,13 @@
+# Configures AWS App public URL. This URL should be accessible by
+# AWS SNS Topic, it can be a public URL or a URL accessible by SNS
+# within the same VPC.
+sns-app-public-url=http://8c349aab.ngrok.io
```

Comment:
Are we ok to have this _public_?

Reply:
Currently, It points to my machine for Dev. and testing. and this one not static . so it may change in future. we need to think on how can we make a stable url for testing with specific SNS Topic settings.